protocol/storage/miner_test.go

```diff
@@ -491,12 +523,12 @@ func testPaymentVouchers(porcelainAPI *minerTestPorcelain, voucherInterval int,
 
 }
 
-func testSignedDealProposal(porcelainAPI *minerTestPorcelain, vouchers []*paymentbroker.PaymentVoucher, addr address.Address) *storagedeal.SignedDealProposal {
+func testSignedDealProposal(porcelainAPI *minerTestPorcelain, vouchers []*paymentbroker.PaymentVoucher, size uint64) *storagedeal.SignedDealProposal {
 	proposal := &storagedeal.Proposal{
 		MinerAddress: porcelainAPI.targetAddress,
 		PieceRef:     types.NewCidForTestGetter()(),
-		TotalPrice:   types.NewAttoFILFromFIL(2500),
-		Size:         types.NewBytesAmount(1000),
+		TotalPrice:   types.NewAttoFILFromFIL(2500 * uint64(math.Ceil(float64(size)/float64(1016)))),
```

Comment:
Why is TotalPrice = 2500 * size / sector size (please add a comment)? Also 1016 should be a variable named sectorSize instead of magic number. Also (2500*size)/1016 is more accurate and more concise unless you need it to be a multiple of 2500 for some reason.

Reply:
I changed the derivation of total price to have no more magic numbers, since there aren't supposed to be any.